shared-library/stos-spring-boot-autoconfigure/src/main/java/com/stos/concert/event/EventProducer.java

```diff
@@ -0,0 +1,17 @@
+package com.stos.concert.event;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Component;
+
+@RequiredArgsConstructor
+@Component
+public class EventProducer {
+
+  private final KafkaTemplate<String, String> kafkaTemplate;
```

Comment:
Template 으로 보내는 payload 를 인터페이스 처리 하면 좋을 듯

spring integration에서 [message의 인터페이스](https://godekdls.github.io/Spring%20Integration/message/)가 잘 정의 되어 있으니까 그걸 활용하면 좋을 것 같음.

 나중에 Header 에 파티션키 같은거도 커스텀 하기 쉬울 것 같고


```suggestion
  private final KafkaTemplate<String, Message<DomainEvent>> kafkaTemplate;
```